lib/cocoapods/target/pod_target.rb

```diff
@@ -398,10 +398,13 @@ def framework_paths
     def resource_paths
       @resource_paths ||= begin
         file_accessors.each_with_object({}) do |file_accessor, hash|
-          resource_paths = file_accessor.resources.map { |res| "${PODS_ROOT}/#{res.relative_path_from(sandbox.project_path.dirname)}" }
-          resource_paths = [] if file_accessor.spec.non_library_specification? && build_as_framework?
+          resource_paths = if file_accessor.spec.non_library_specification? && build_as_framework?
```

Comment:
minor speed boost here, no need to process `file_accessor.resources` (which hits the file system) if we know we will throw it away.